source/reference/minio-mc/mc-ilm-tier-add.rst

```diff
@@ -0,0 +1,343 @@
+.. _minio-mc-ilm-tier-add:
+
+===================
+``mc ilm tier add``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc ilm tier add
+
+.. versionchanged:: RELEASE.2022-12-24T15-21-38Z
+
+   :mc-cmd:`mc ilm tier add` replaces ``mc admin tier add``.
+
+Description
+-----------
+
+.. start-mc-ilm-tier-add-desc
+
+The :mc:`mc ilm tier add` command creates a new remote storage tier to a supported S3-compatible service for MinIO :ref:`Lifecycle Management: Object Transition ("Tiering")<minio-lifecycle-management-expiration>`. 
```

Comment:
Azure blob is actually not S3 compatible at all, and some of the others are only sort of compatible.

We can probably sub this to just "supported storage services. See <x> for a complete list"

Reply:
Done